gdnative/multi-lod/src/lod.h

```diff
@@ -188,18 +191,22 @@ class LOD : public VisualInstance, public LODBaseVariables {
 
     bool disable_processing = false;
 
-    // Keep track of last state to avoid unnecessary show/hide/process toggle calls
-    int last_state = -1;
-
     NodePath lod0_path;
     NodePath lod1_path;
     NodePath lod2_path;
     NodePath lod3_path;
 
-    Spatial* lod0 = NULL; // LOD0 MUST exist otherwise screen percentage breaks
-    Spatial* lod1 = NULL; // Plus it doesn't make sense for it not to exist
-    Spatial* lod2 = NULL;
-    Spatial* lod3 = NULL;
+    // LOD0 MUST exist otherwise screen percentage breaks
+    // Plus it doesn't make sense for it not to exist
+    Spatial* lods[LOD_COUNT] = { };
+
+    int last_lod = 0;   // Last active LOD index
```

Comment:
Can you move this comment to be above the variable and reword it to "The highest level of LOD available/found."